app/views/layouts/application.html.erb

```diff
@@ -19,7 +19,9 @@
     <div class="container p-3 my-2 bg_dark text-black">
       <nav class="navbar navbar-expand-lg navbar-light bg-light">
         <%= link_to 'Articles', root_path, class: "navbar-brand" %>
-        <%= link_to 'Admin', users_path, class: "navbar-brand", admin: true %>
+        <% if current_user.role =='admin'%>
```

Comment:
I think I told 3 times to use the instance method from `app/models/user.rb`

Reply:
I updated this file but It was staged mistakenly.

Reply:
It was staged mistakenly.

Reply:
I have updated this file.